### PR TITLE
Decision 17754 upgrade breadcrumb

### PR DESCRIPTION
**Pull request by Dekel-Cohen**

### CODE REVIEW

File: apps/demo/src/app/app.component.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "breadcrumb/breadcrumb-theme";
+@import "../../../../libs/angular-breadcrumb/src/breadcrumb/breadcrumb-theme";
```
Comment (guy-eizon) [inline apps/demo/src/app/app.component.less]: How come the import was needed to be changed? Was the structure changed?

File: apps/demo/src/app/app.component.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {Breadcrumb} from "../../../../libs/angular-breadcrumb/src/common/model/b
 @Component({
   selector: 'app-root',
   templateUrl: 'app.component.html',
-  styleUrls: ['app.component'],
+  styleUrls: ['app.component.less'],
```
Comment (guy-eizon) [inline apps/demo/src/app/app.component.ts]: Doesn't it expect `css` extension?

File: apps/demo/src/polyfills.ts [hunk 3]
```diff
@@ -42,7 +42,7 @@
 
 /** Evergreen browsers require these. **/
 import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
```
Comment (guy-eizon) [inline apps/demo/src/polyfills.ts]: 👍

File: package.json [hunk 4]
```diff
@@ -23,30 +23,33 @@
...
     "@ngrx/effects": "4.1.1",
     "@ngrx/router-store": "4.1.1",
     "@ngrx/store": "4.1.1",
     "@ngrx/store-devtools": "4.0.0",
     "@nrwl/nx": "0.6.19",
     "core-js": "^2.4.1",
     "less": "^3.0.1",
-    "rxjs": "^5.5.2",
+    "rxjs": "^6.4.0",
+    "rxjs-compat": "^6.4.0",
```
Comment (guy-eizon) [inline package.json]: What about removing `rxjs-compat`?

File: tsconfig.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "importHelpers": true,
```
Comment (guy-eizon) [inline tsconfig.json]: Can you explain for my knowledge what is it for?

File: tsconfig.json [hunk 6]
```diff
@@ -16,11 +17,14 @@
     ],
     "baseUrl": ".",
     "paths": {
-      "angular-breadcrumb": ["libs/angular-breadcrumb"]
-    }
+      "angular-breadcrumb": [
+        "libs/angular-breadcrumb"
+      ]
+    },
+    "module": "es2015"
```
Comment (guy-eizon) [inline tsconfig.json]: Can you explain for my knowledge what is it for?

File: apps/demo/src/app/app.component.less (hunk 1 above)
Comment (Dekel-Cohen) [reply apps/demo/src/app/app.component.less]: We are not sure it ever worked before.
The important thing is to make it work now, and the path for the import was incorrect

File: apps/demo/src/app/app.component.ts (hunk 2 above)
Comment (Dekel-Cohen) [reply apps/demo/src/app/app.component.ts]: no, it expects whatever you define in ```angular-cli```

File: package.json (hunk 4 above)
Comment (Dekel-Cohen) [reply package.json]: will check

File: tsconfig.json (hunk 5 above)
Comment (Dekel-Cohen) [reply tsconfig.json]: good questions, @rantzv  did you manually change it or was it automatically generated by the upgrade somehow?

File: package.json (hunk 4 above)
Comment (Dekel-Cohen) [reply package.json]: removed

File: apps/demo/src/app/app.component.less (hunk 1 above)
Comment (guy-eizon) [reply apps/demo/src/app/app.component.less]: 👍

File: apps/demo/src/app/app.component.ts (hunk 2 above)
Comment (guy-eizon) [reply apps/demo/src/app/app.component.ts]: 👍

File: package.json [hunk 7]
```diff
@@ -40,8 +40,7 @@
     "core-js": "^2.4.1",
     "less": "^3.0.1",
     "rxjs": "^6.4.0",
-    "rxjs-compat": "^6.4.0",
```
Comment (guy-eizon) [inline package.json]: Did you uninstall it and checked that it is working?

File: tsconfig.json (hunk 5 above)
Comment (guy-eizon) [reply tsconfig.json]: @rantzv any inputs?

File: package.json (hunk 7 above)
Comment (Dekel-Cohen) [reply package.json]: yes

File: tsconfig.json (hunk 5 above)
Comment (rantzv) [reply tsconfig.json]: @guy-eizon @Dekel-Cohen This is the result of the `ng update @angular/cli --migrate-only --from=5.1` command

**Pull request merged**